### DISCUSSION TITLE
Pay what you want Ticket. Additional Product included at a specific threshold

### DISCUSSION THREAD

**Discussion by HDValentin** [pretix/pretix#1968] [Ideas]
We have the pay what you can/want tickets at our barcamp.
Is it possible to create a functionality that at a threshold a T-Shirt is inclusive?

For example
Person 1 buys a ticket and gives 25 Euro.
The T-Shirt has a price and is to buy if the person want one.

Person 2 buys a ticket and gives 90 Euro.
The T-Shirt has no price and he/she can add it as additional product.

If this is no feature at the moment. Would you implement that?
Or is this possible to implement it as a plugin?

**Comment by felixrindt** [comment]
As far as I understand, this would not provide new functionality, as you could have the shirt as a standalone product. Someone could then decide to either pay 90 Euro and get no T-Shirt or include the T-Shirt and just pay (90 Euro minus the Shirt price) for the tickets.

If you want some sort of discount, this might be related to #433 and #1001.

**Comment by HDValentin** [comment]
You can add aditional Products to the Ticket. But it is only for free or for pay. See attached Screenshot.
There ist no option "if payed price is higher than X" than give it for free.
![zwischenablage01](https://user-images.githubusercontent.com/1276955/50055354-008f9f00-014e-11e9-8d1f-bb0d50b71986.jpg)

**Comment by felixrindt** [comment]
You don't want to have the shirt as a standalone product, because it you want it to be bought in combination with a ticket. Also, you don't want to have separate ticket products (with and without T-Shirt as an addon). 

In your example, let's say a shirt costs 20 Euro, why shouldn't Person 2 be able to buy the ticket for 70 Euro and buy the T-Shirt for an additional 20?

Please don't feel offended, but Pretix already has many options and settings, so a new feature should have a clear reason to exist.

**Comment by HDValentin** [comment]
I've attached the this year ticketshop screenshot. I'll hope it is getting clearer.
We don't want to make it more complicated as it is.
So we don't want to create a normal ticket pay what you want under 50 Euro 
and a normal ticket pay what you want over 50 Euro where a Shirt is included.

![tickets-shirts](https://user-images.githubusercontent.com/1276955/50058131-6727b380-0174-11e9-855d-a1a2e0b74730.jpg)

**Comment by raphaelm** [comment]
Hmm, I get why this is useful, but I share Felix' concern. This is a really edge-casy option, and I don't really see that it would be *so* bad to have a second type of "Förderticket" to handle this.

**Comment by HDValentin** [comment]
OK. Please close this feature request If it is not easy to implement.
Anyway thanks for your great work!
